packages/web/server/index.ts

```diff
@@ -20,6 +21,7 @@ import { getFormattedMediumArticles } from './mediumAPI'
 const port = parseInt(process.env.PORT, 10) || 3000
 
 const dev = process.env.NEXT_DEV === 'true'
+// tslint:disable-next-line
```

Comment:
What is wrong with this?

Reply:
ah these must have been left over from a trip down a different rabbit hole